invenio_mail/ext.py

```diff
@@ -67,5 +67,11 @@ def init_config(self, app):
     def print_email(self, message, app):
         """Print mail to stream."""
         with self._lock:
-            self.stream.write("{0}\n{1}\n".format(message.as_string(), '-'*79))
-            self.stream.flush()
+            invenio_mail = app.extensions['invenio-mail']
```

Comment:
@JavierDelgadoFernandez it's maybe the easiest but I don't see why did you make this change.

One can always do the same with: `app.extensions['invenio-mail'].print_email(message)`